lib/wallet.js

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback, useContext, useMemo } from 'react'
+import env from './environment'
+import { UnsupportedChainError, UseWalletProvider, useWallet } from 'use-wallet'
+import { providers } from 'ethers'
+import { getNetworkName } from './web3-utils'
+import { trackEvent } from 'lib/analytics'
+
+const CHAIN_ID = Number(env('CHAIN_ID'))
+const WalletAugmentedContext = React.createContext()
```

Comment:
This is a bit of a nitpick, but I think we can leave a line in between these, as they're not related.